internal/resolve/catalog_test.go

```diff
@@ -935,3 +940,26 @@ func TestMultiplePriority(t *testing.T) {
 	assert.Nil(t, gotVersion)
 	assert.Nil(t, gotDeprecation)
 }
+
+func TestMultipleChannels(t *testing.T) {
```

Comment:
We should make some change to ensure there's a different outcome between:
1. leave channels empty
2. set both `alpha` and `beta`

Maybe we should add a `gamma` channel in `genPackage` that has a `3.0.0` version in it. That way there would be different outcomes in the above cases:
1. select 3.0.0
2. select 2.0.0